app/views/pages/home.html.erb

```diff
@@ -1,4 +1,4 @@
 <h1>Pages#home</h1>
 <p>
-  <%= current_user.email if user_signed_in? %>
+  <%= current_user.username || current_user.email if user_signed_in? %>
```

Comment:
Change to ternary operator tks

Reply:
ok boss. done. please tell me if can merge not